content/articles/blog/drizzle.md

```diff
@@ -0,0 +1,227 @@
+	failures, in other words I have no skin in the game. In addition,
+	I was never privy to any information about Drizzle except that
+	which became available via public channels, so I am not at risk of
+	divulging anything private or confidential.
+
+Prior to that, the project had been mostly dormant since 2012[^2013],
+having been founded in 2008. So it was properly "active" for just 4
+years, and then in limbo for 4 more before finally wrapping
+up. Chances are, you've probably never run a Drizzle database server
+in production, and quite possibly never spun one up for any purpose
+either.
+
+[^2013]: The project did participate in Google Summer of Code in 2013,
+    which is what the [last tweets on the project's Twitter
+    account](https://twitter.com/drizzledb) are about. But the
+    project's development branch [had its last alpha release in
+    September 2012](https://launchpad.net/drizzle/7.2).
+
+And yet, if you're an open source software developer, you're probably
+using something, every single day, that came out of Drizzle. And that
+something isn't even software.
+
+## Drizzle's history, a very brief summary
+
+Drizzle started as an attempt to refactor MySQL, and was originally
+driven by Brian Aker, together with a small team of engineers at Sun
+(which had then-recently acquired MySQL), in the first half of 2008. A
+skunk works project that flew under the radar — to put it charitably —
+at Sun, Drizzle was publicly announced at O'Reilly OSCON of that
+year. There are a couple of videos floating around from that event
+([from the keynotes](https://youtu.be/9DuJFUnxg7k), and [from a booth
+presentation](https://youtu.be/2tO7_Ozr-9U)) that are both... well, go
+and see for yourself. The aforementioned Stewart Smith did a very
+entertaining talk at linux.conf.au some five years later that covers
+those events, which you can watch from [the official Linux Australia
+mirror](https://mirror.linux.org.au/pub/linux.conf.au/2014/Wednesday/28-Past_Present_and_future_of_MySQL_and_variants_-_Stewart_Smith.mp4),
+or from a [YouTube
+upload](https://www.youtube.com/watch?v=6Uv9vcb4SdA).
+
+There's also an interesting old blog post from MySQL co-founder Monty
+Widenius, [written in late July of
+2008](https://monty-says.blogspot.com/2008/07/what-if.html), which
+outlines the state of affairs at the time.
+
+Of course, in 2010 Oracle acquired Sun (and with it, the MySQL
+database) — and Oracle was presumably less than keen on having an
+*in-house* fork of the database technology it had just acquired. Thus,
+the Drizzle engineers found a new home at Rackspace, with the goal of
+getting Drizzle to a production-ready release. That sort of happened,
+and the Drizzle package even got into Debian, but after [the Drizzle
+7.1 release in 2012](https://launchpad.net/drizzle/7.1), adoption did
+not exactly skyrocket. Development on Drizzle stagnated and eventually
+petered out. The 7.2 release branch [never made it out of the alpha
```

Comment:
mainly because everybody quickly got other jobs when told they were no longer going to have one, and we all just didn't have the capacity to work full time on it after hours.